go/libkb/naclwrap.go

```diff
@@ -860,3 +879,13 @@ func (k NaclDHKeyPair) Decrypt(nei *NaclEncryptionInfo) (plaintext []byte, sende
 	sender = senderDH.GetKID()
 	return
 }
+
+func GeneratePerUserKeySeed() (PerUserKeySeed, error) {
+	var seed [PerUserKeySeedSize]byte
+	if nRead, err := rand.Read(seed[:]); err != nil {
+		return seed, err
+	} else if nRead != len(seed) {
+		return seed, fmt.Errorf("Short random read: %d", nRead)
+	}
+	return seed, nil
+}
```

Comment:
I feel like we have a few versions of this floating around. Would it be simpler to do `PerUserKeySeed(MakeByte32(RandBytes(32)))`?